src/components/Button.js

```diff
@@ -58,7 +56,8 @@ class Button extends React.Component {
   };
 
   render () {
-    const styles = this.styles();
+    const theme = StyleUtils.mergeTheme(this.props.theme, this.props.primaryColor);
```

Comment:
eventually i think we could kill `this.props.primaryColor` and just use the theme